tests/acceptance/search/search-help-test.ts

```diff
@@ -0,0 +1,100 @@
+        assert.dom('[data-test-start-help]').exists();
+        await click('[data-test-start-help]');
+
+        // verify first popover displays
+        assert.dom('[data-test-search-help-1]').exists();
+        // verify skip button present
+        assert.dom('[data-test-help-skip-1]').exists();
+        assert.dom('[data-test-help-skip-1]').hasText(this.intl.t('search_help_test.skip'));
+        assert.dom('[data-test-help-skip-1]').hasAttribute('type', 'reset');
+        // verify next button works
+        assert.dom('[data-test-help-next-1]').exists();
+        assert.dom('[data-test-help-next-1]').hasText(this.intl.t('search_help_test.next'));
+        assert.dom('[data-test-help-next-1]').hasAttribute('type', 'submit');
+        // verify first popover content
+        assert.dom('[data-test-help-heading-1]').exists();
+        assert.dom('[data-test-help-heading-1]').hasText(this.intl.t('search_help_test.heading_1_test'));
+        assert.dom('[data-test-help-body-1]').exists();
+        assert.dom('[data-test-help-body-1]').hasText(this.intl.t('search_help_test.body_1_test'));
+        assert.dom('[data-test-help-enumeration-1]').exists();
+        assert.dom('[data-test-help-enumeration-1]').hasText(this.intl.t('search_help_test.enumeration_1_test'));
+
+        // verify popover position and spacing
+        // assertTooltipSpacing(assert, 20);
+        // assertTooltipSide(assert, 'bottom');
+
+        // verify second popover displays
+        await click('[data-test-help-next-1]');
+        assert.dom('[data-test-search-help-2]').exists();
+        // verify second popover content
+        assert.dom('[data-test-help-heading-2]').exists();
+        assert.dom('[data-test-help-heading-2]').hasText(this.intl.t('search_help_test.heading_2_test'));
+        assert.dom('[data-test-help-body-2]').exists();
+        assert.dom('[data-test-help-body-2]').hasText(this.intl.t('search_help_test.body_2_test'));
+        assert.dom('[data-test-help-enumeration-2]').exists();
+        assert.dom('[data-test-help-enumeration-2]').hasText(this.intl.t('search_help_test.enumeration_2_test'));
+
+        // verify third popover displays
+        await click('[data-test-help-next-2]');
+        assert.dom('[data-test-search-help-3]').exists();
+        // verify third popover content
+        assert.dom('[data-test-help-heading-3]').exists();
+        assert.dom('[data-test-help-heading-3]').hasText(this.intl.t('search_help_test.heading_3_test'));
+        assert.dom('[data-test-help-body-3]').exists();
+        assert.dom('[data-test-help-body-3]').hasText(this.intl.t('search_help_test.body_3_test'));
+        assert.dom('[data-test-help-enumeration-3]').exists();
+        assert.dom('[data-test-help-enumeration-3]').hasText(this.intl.t('search_help_test.enumeration_3_test'));
+
+        // verify popover closes
+        await click('[data-test-help-done]');
+        assert.dom('[data-test-search-help-1]').isNotVisible();
```

Comment:
This will fail with doesNotExist() because the element is obscured visibly with ariaHidden = true but still is sibling to the adjacent element.